community-containers/jellyseerr/jellyseerr.json

```
@@ -0,0 +1,37 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-jellyseerr",
+            "display_name": "Jellyseerr",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/jellyfin",
+            "image": "fallenbagel/jellyseerr",
+            "internal_port": "5055",
+            "restart": "unless-stopped",
+            "ports": [
+                {
+                    "ip_binding": "",
+                    "port_number": "5055",
+                    "protocol": "tcp"
+                },
+                {
+                    "ip_binding": "",
+                    "port_number": "5055",
+                    "protocol": "udp"
+                }
```

I think jellyseerr only needs tcp based on https://docs.jellyseerr.dev/getting-started/docker?docker-methods=docker-compose

```suggestion
                }
```